docs/tutorials/07_pegasos_qsvc.ipynb

```diff
@@ -19,59 +23,185 @@
    "id": "thirty-painting",
    "metadata": {},
    "source": [
-    "Let's generate some data:"
+    "### 1. Preparing dataset"
```

Comment:
I would add a section with "the rest of the tutorial is structured as follows": index

Reply:
sure